castervoice/rules/apps/editor/vscode_rules/vscode2.py

```diff
@@ -75,10 +75,12 @@ class VSCodeNonCcrRule(MappingRule):
         # File Management
         "copy path":
             R(Key("c-k, p")),
-        "[open] command palette":
-            R(Key("cs-p"), rdescript="VS Code: Command Palette"),
+        "[open] command palette [<text>]":
```

Comment:
We might as well add `Pause("30")` can at the same raising condition as open project.